stage0/src/lib.rs

```diff
@@ -252,8 +258,12 @@ pub fn rust64_start(encrypted: u64) -> ! {
         entry = VirtAddr::new(header.e_entry);
     }
 
-    let rsdp = acpi::build_acpi_tables(&mut fwcfg).unwrap();
+    let mut acpi_digest = Sha256::default();
+    let rsdp = acpi::build_acpi_tables(&mut fwcfg, &mut acpi_digest).unwrap();
```

Comment:
Is Stage0 creating ACPI tables?

Reply:
Sort-of. We get files and command from the VMM via the fw_cfg device to construct them. We don't build them fully.